R/dock_from_desc.R

```diff
@@ -36,7 +36,9 @@ base_pkg_ <- c(
 #' @param FROM The FROM of the Dockerfile. Default is
 #'     FROM rocker/r-ver:`R.Version()$major`.`R.Version()$minor`.
 #' @param AS The AS of the Dockerfile. Default it NULL.
+#' @param sha256 character. The Digest SHA256 hash corresponding to the chip architecture of the deployment host machine if different than the machine on which the image will be built.
```

Comment:
Could you please elaborate on this parameter `sha256` ?  
How is this different from direct use of:
```r
dock_from_desc(
  FROM = rocker/rver@sha256:xxxxxxxxx
)
```
If this is really needed, could you add information on how to create it and why in a "Details" section ?

Reply:
Hey @statnmap, this is no different from the direct use as you've indicated. It's provided as an argument to prompt a user that might not know they need to specify the build architecture when the host machine is different from the build machine. I've added better documentation in a7c3658f78a06f8023019be88eb15750abf486ad and added the argument to `docker_from_renv` for parity across the two functions.